src/script/user/UserRepository.ts

```diff
@@ -105,7 +105,16 @@ interface UserAvailabilityEvent {
   type: USER.AVAILABILITY;
 }
 
-type Events = {supportedProtocolsUpdated: {user: User; supportedProtocols: ConversationProtocol[]}};
+type Events = {
+  supportedProtocolsUpdated: {user: User; supportedProtocols: ConversationProtocol[]};
+  userDeleted: QualifiedId;
+};
+
+type UserDeleteEventWithQualifiedId = {
+  id: string;
+  qualified_id: QualifiedId;
+  type: USER_EVENT.DELETE;
+};
```

Comment:
the `UserDelete` Event form backend has the correct shape but we didn't add the `qualified_id` field to its type to the web-packages. This is addressed here https://github.com/wireapp/wire-web-packages/pull/7252 but we do not have the option to bump the `core` package in the `release/q2-2025` branch.

This is a work around to extend and cast the correct type